### DISCUSSION TITLE
Technical Talk on Cignas Implementation of the Price Transparency Files

### DISCUSSION THREAD

**Discussion by gardnmi** [CMSgov/price-transparency-guide#595] [General]
[Journey to Solving Healthcare Price Transparency with Databricks and Delta Lake](https://www.youtube.com/watch?v=YfCjMlHlZCg)

As someone who is trying to consume this data it is a good insight to see how they are producing the data and why there are issues that make this data less mrf than intended.

**Comment by dgolden** [comment]
Fortunately Keyark.com can ingest and produce analytics faster than Cigna can generate these MRF!

Great quotes in the video:  "CMS In-network Rates MRF Schema is not Spark friendly" and "Even Spark has its limits"

**Reply by gardnmi** [reply]
I'll need to explore keyark but I've spent weeks of processing time just flattening these files to prep for spark ingestion.

**Reply by dgolden** [reply]
@gardnmi  My email address is found in the dgolden profile. Perhaps I can help. I was wondering how easy it might be to ingest using Spark ... sounds like not easy at all.

**Comment by tjmc** [comment]
Does someone have a mirror of the video? It looks like it was taken down.

**Comment by dgolden** [comment]
Here is a screen capture from the video that shows its origin:

![image](https://user-images.githubusercontent.com/6385204/191163688-d06a1b13-8ce8-49fb-9ea1-a49c4b727ebb.png)
